sqlite/migrate.go

```diff
@@ -18,7 +19,7 @@ type PostStep struct {
 	RollBackVersion uint
 }
 
-var migrationTable = "status_go_" + sqlcipher.DefaultMigrationsTable
+var MigrationTableName = "status_go_" + sqlcipher.DefaultMigrationsTable
```

Comment:
Since it's a `var`, better keep it private and add a getter

Reply:
Looking again, perhaps we should just move `MigrationTableName` to the `appdatabase` package?

This `sqlite` is a generic utils package to work with SQLite databases, it shouldn't really define the default migration table name. While `appdatabase.MigrationTableName` makes complete sense.

Reply:
It's kinda weird that `appdatabase` asks for `sqlite.MigrationTableName`:
https://github.com/status-im/status-go/blob/6445824157eddb9fae640b2233984526d32adc65/appdatabase/database.go#L46-L47

Reply:
Changed to:
```golang
func StatusMigrationTableName() string {
	return "status_go_" + sqlcipher.DefaultMigrationsTable
}
```